lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -73,12 +76,36 @@ public function register_routes() {
 				'callback' => array( $this, 'update_item' ),
 				'permission_callback' => array( $this, 'update_item_permissions_check' ),
 				'args'     => array(
-					'post'         => array(),
-					'status'       => array(),
-					'content'      => array(),
-					'author'       => array(),
-					'author_email' => array(),
-					'author_url'   => array(),
+					'post'         => array(
```

Comment:
@joehoyle Hmm...another issue I can see with using the Schema is that on comment update you can change the 'status' (which you cannot set on create), so that field isn't completely `readonly`.  What do you think we should do here?